FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+Search.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ComposeViewController+Search.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 4/6/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+// MARK: - Search
+extension ComposeViewController {
+    internal func setupSearch() {
+        search
+            .debounce(for: .milliseconds(300), scheduler: RunLoop.main)
+            .removeDuplicates()
+            .map { [weak self] query -> String in
+                if query.isEmpty {
+                    self?.updateState(with: .main)
+                }
+                return query
+            }
+            .sink(receiveValue: { [weak self] in
+                guard $0.isNotEmpty else { return }
+                self?.searchEmail(with: $0)
```

Comment:
Let's put this in `SetupUI` which can be renamed to `Setup`

Reply:
Yeah, I agree